src/i18n/fi/translation.json

```diff
@@ -174,7 +174,8 @@
     "index": "Omat sivuni",
     "preferences": "Asetukseni",
     "something-else": "Jotain muuta",
-    "work-history": "Työpaikkani"
+    "work-history": "Työpaikkani",
+    "help-text": "Rakenna koulutus- ja työhistoriasi käyttämällä alla olevia työkaluja, jolloin palvelu voi tunnistaa sinulle sopivia osaamisia."
```

Comment:
Just a small thing, but we try to keep the keys in alphabetical order.
So in this case `"help-text"` to be above `"index"`